README.md

```diff
@@ -37,6 +38,29 @@ It also provides the `crud-storage` and `crud-router` roles for
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Quickstart
+
+First, [install Tarantool](https://www.tarantool.io/en/download).
+
+Now you have the following options how to learn crud API and use it in a
+project:
+
+* Play with crud on a testing dataset on a single instance:
+
+  ```shell
+  $ git clone https://github.com/tarantool/crud.git
+  $ cd crud
+  $ tarantoolctl rocks install vshard
```

Comment:
$ tarantoolctl rocks install vshard
$ doc/playground.lua 
LuajitError: /home/ozi/tmp/crud/crud/cfg.lua:5: module 'checks' not found:
	no field package.preload['checks']
	no file '/home/ozi/tmp/crud/checks.lua'
	no file '/home/ozi/tmp/crud/checks/init.lua'
	no file '/home/ozi/tmp/crud/checks.so'
	no file '/home/ozi/tmp/crud/.rocks/share/tarantool/checks.lua'
	no file '/home/ozi/tmp/crud/.rocks/share/tarantool/checks/init.lua'
	no file '/home/ozi/tmp/.rocks/share/tarantool/checks.lua'
	no file '/home/ozi/tmp/.rocks/share/tarantool/checks/init.lua'
	no file '/home/ozi/.rocks/sha
fatal error, exiting the event loop
```suggestion
  $ ./deps.sh
```

Reply:
`deps.sh` installs everything that is needed for development. Here I had idea to install only what is needed for using crud. But, yep, I missed that we need checks and errors.

`tarantoolctl rocks make` here would be more suitable, I think.

Reply:
Replaced `tarantoolctl rocks install vshard` with `tarantoolctl rocks make`.

Reply:
It works, thank you!